src/Hazelcast.Net/DistributedObjects/IHRingBuffer.cs

```diff
@@ -211,7 +211,7 @@ public interface IHRingBuffer<TItem> : IDistributedObject
         /// +1.
         /// </exception>
         /// <exception cref="System.Exception">if the call is interrupted while blocking.</exception>
-        ValueTask<TItem> GetAsync(long sequence);
+        ValueTask<TItem> ReadOneAsync(long sequence);
```

Comment:
Not specific to this one, but there are documentation showing the old style of the client like 
```
        /// Ringbuffer&lt;String&gt; ringbuffer = hz.GetRingbuffer("rb");
        /// long seq = ringbuffer.HeadSequence();
        /// while(true){
        /// String item = ringbuffer.ReadOne(seq);
        /// seq++;
        /// ... process item
        /// }
```
It would be good to update them

Reply:
This can be done later, approving the PR as it is. Maybe it would be good to create an issue to track these stale documentations